src/codegen/llvm/intrin_rule_rocm.cc

```diff
@@ -0,0 +1,48 @@
+/*!
+ *  Copyright (c) 2017 by Contributors
+ * \file intrin_rule_llvm.cc
+ */
+#ifdef TVM_LLVM_VERSION
+
+#include "./intrin_rule_llvm.h"
+#include <tvm/ir.h>
+#include <tvm/expr.h>
+#include <tvm/api_registry.h>
+#include <map>
+
+namespace tvm {
+namespace codegen {
+
+inline void DispatchExternOCML(const TVMArgs& args, TVMRetValue* rv) {
+  Expr e = args[0];
+  using namespace ir;
+  const Call* call = e.as<Call>();
+  CHECK(call != nullptr);
+  const std::string bit_width = std::to_string(call->type.bits());
```

Comment:
it might be easier to use std::ostringstream to construct the string

Reply:
specifically use


```
std::ostringstream intrin_name;

intrin_name << "__ocml_" << call->name << "_f" << call->type.bits();
// use intrin_name.str()
```

Reply:
yes, it is fixed